prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/model/VectorMultimapUtils.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.prinz.pmml.model
+
+import pl.touk.nussknacker.prinz.util.collection.immutable.VectorMultimap
+
+object VectorMultimapUtils {
```

Comment:
maybemove this to util package in main (not pmml) module?

Reply:
Can we do this in a separate PR? I don't want to touch MLflow in this PR.